### PR TITLE
Issue 43

### PR DESCRIPTION
**Pull request by weaglesBio**
## PR checklist

- [X] This comment contains a description of changes (with reason).
- [X] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/treeval/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/treeval _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [X] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

Replaced insilico digest and splitfasta sanger-tol modules with local.

### CODE REVIEW

**Review by muffato — COMMENTED** [review]
It's great that you could find an existing container with BioPerl. Overall, I think it materially simplifies the pipeline 👍🏼  . As in #44, it'd be perfect if you could version all four scripts and remove the executable bit of the `.nf` file

**Review by priyanka-surana — COMMENTED** [review]
`yaml_input.nf` is an empty file. Maybe delete it if it is not needed.

**Comment by weaglesBio** [comment]
Added credit, version and made non-executable. Tested locally.

File: modules/local/makecmap_cmap2bed.nf [hunk 1]
```diff
@@ -2,13 +2,10 @@ process MAKECMAP_CMAP2BED {
     tag "$meta.id"
     label 'process_medium'
```
Comment (muffato) [inline modules/local/makecmap_cmap2bed.nf:3]: I believe the script is single-threaded
```suggestion
    label 'process_single'
```

File: modules/local/makecmap_fa2cmapmulticolor.nf [hunk 2]
```diff
@@ -2,12 +2,10 @@ process MAKECMAP_FA2CMAPMULTICOLOR {
     tag "$meta.id"
     label 'process_medium'
```
Comment (muffato) [inline modules/local/makecmap_fa2cmapmulticolor.nf:3]: I believe the script is single-threaded
```suggestion
    label 'process_single'
```

File: modules/local/makecmap_renamecmapids.nf [hunk 3]
```diff
@@ -2,12 +2,10 @@ process MAKECMAP_RENAMECMAPIDS {
     tag "$meta.id"
     label 'process_medium'
```
Comment (muffato) [inline modules/local/makecmap_renamecmapids.nf:3]: I believe the script is single-threaded
```suggestion
    label 'process_single'
```

File: modules/local/selfcomp_mapids.nf [hunk 4]
```diff
@@ -1,27 +1,33 @@
 process SELFCOMP_MAPIDS {
     tag "$meta.id"
     label 'process_medium'
```
Comment (muffato) [inline modules/local/selfcomp_mapids.nf:3]: I believe the script is single-threaded
```suggestion
    label 'process_single'
```

File: modules/local/selfcomp_mummer2bed.nf [hunk 5]
```diff
@@ -1,11 +1,11 @@
 process SELFCOMP_MUMMER2BED {
     tag "$meta.id"
     label 'process_medium'
```
Comment (muffato) [inline modules/local/selfcomp_mummer2bed.nf:3]: I believe the script is single-threaded
```suggestion
    label 'process_single'
```

File: modules/local/selfcomp_splitfasta.nf [hunk 6]
```diff
@@ -2,12 +2,10 @@ process SELFCOMP_SPLITFASTA {
     tag "$meta.id"
     label 'process_medium'
```
Comment (muffato) [inline modules/local/selfcomp_splitfasta.nf:3]: I believe the script is single-threaded
```suggestion
    label 'process_single'
```

**Review by muffato — COMMENTED** [review]
Can you remove the executable permission of `subworkflows/local/yaml_input.nf` and `workflows/treeval.nf` ?

File: bin/cmap2bed.py [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+import io
+import string
+
+<<<<<<< HEAD
+parser = optparse.OptionParser()
+=======
+# Script originally developed by Yumi Sims (yy5@sanger.ac.uk)
+
+parser = optparse.OptionParser(version="%prog 1.0")
+>>>>>>> f3007388302cde3cecfe7a50820849432b0c8b12
```
Comment (muffato) [inline bin/cmap2bed.py:14]: There's a merge conflict here. Can you also make sure the script supports `--version` ?

File: bin/rename_cmapids.pl [hunk 8]
```diff
@@ -0,0 +1,218 @@
+#! /usr/local/bin/perl
+<<<<<<< HEAD
+=======
+
+# Script originally developed by Yumi Sims (yy5@sanger.ac.uk)
+
+>>>>>>> f3007388302cde3cecfe7a50820849432b0c8b12
```
Comment (muffato) [inline bin/rename_cmapids.pl:7]: Merge conflict

File: bin/split_genomes_for_ensembl.pl [hunk 9]
```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env perl
+
+<<<<<<< HEAD
+=======
+# Script originally developed by Yumi Sims (yy5@sanger.ac.uk)
+
+>>>>>>> f3007388302cde3cecfe7a50820849432b0c8b12
```
Comment (muffato) [inline bin/split_genomes_for_ensembl.pl:7]: Merge conflict

**Review by muffato — COMMENTED** [review]
Just a few merge conflicts

**Review by muffato — APPROVED** [review]

**Pull request merged**